### ISSUE TITLE
devServer.before and devServer.after method does not exist

### ISSUE THREAD

**Issue by nashaofu** [neutrinojs/webpack-chain#164]
devServer has no before and after methods. I made a mistake when I was using it. thanks！
https://webpack.js.org/configuration/dev-server/#devserverbefore

**Comment by eliperelman** [comment]
When a method doesn't exist, you can manually set it via `.set`:

```js
config.devServer.set('before', () => {});
config.devServer.set('after', () => {});
```

We would welcome a PR to update those methods.

**Comment by eliperelman** [comment]
Fixed by #167.
